src/ensembl/src/content/app/browser/browser-image/BrowserImage.test.tsx

```diff
@@ -25,6 +25,15 @@ import { createMockBrowserState } from 'tests/fixtures/browser';
 
 import { BrowserImage } from './BrowserImage';
 
+import MockGenomeBrowser from 'tests/mocks/mockGenomeBrowser';
+
+const mockGenomeBrowser = jest.fn(() => new MockGenomeBrowser() as any);
```

Comment:
out of curiosity, why was the `as any` needed?

Reply:
I'm setting the `mockReturnValue` to `undefined` here:
https://github.com/Ensembl/ensembl-client/pull/507/files/4d4ad016a8abdcaa24a45b8db93bca1b19e5ded1#diff-adff5937799cc55e1c8f56b9f4ab7c845469c307fb41dbcf7a40324b0e08ad65R75

Typescript complains if I don't use `as any` here.

There other option will be is to do the following:
const mockGenomeBrowser : jest.Mock<MockGenomeBrowser | undefined, []> = jest.fn(() => new MockGenomeBrowser());